Assets/SEE/UI/Window/CodeWindow/Input/CodeWindowInput.cs

```diff
@@ -377,69 +376,65 @@ public void EnterFromFile(string filename, bool syntaxHighlighting = true)
             }
         }
 
-        private async UniTaskVoid MarkIssues(string path)
+        private async UniTaskVoid MarkIssuesAsync(string path)
```

Comment:
Documentation for this method is missing.

Reply:
Fixed it.